decidim-budgets/app/cells/decidim/budgets/budget_history_cell.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Budgets
+    # This cell is used to render the budget history panel of a resource
+    # inside a tab of a show view
+    #
+    # The `model` must be a resource to get the budget history from.and is expected to
+    # respond to budget history method
+    #
+    # Example:
+    #
+    #   cell(
+    #     "decidim/budget_history",
+    #     budget
+    #   )
+    class BudgetHistoryCell < Decidim::ViewModel
+      include Decidim::Budgets::ApplicationHelper
+
+      def show
+        render
+      end
+
+      private
+
+      def history_items
```

Comment:
I see that we are using this method publicly, let's just put the private after and use it normally then. This applies to all history cells

Reply:
Done